cli/cli.go

```diff
@@ -103,7 +105,7 @@ USAGE
   anycable-go [options]
 
 OPTIONS
-  --host                   Server host, default: localhost, env: ANYCABLE_HOST
```

Comment:
I feel the necessity to change default instead of the help but it's too dangerous 🤷‍♂️.

Reply:
Mm, then we shouldn't change the help message.

Can we add a deprecation message?

For example:
- use some "fake" value as the default, e.g. (in https://github.com/anycable/anycable-go/blob/85d3347a2e0943e12259c530abb0be2b317f5f3a/cli/cli.go#L50):

```go
fs.StringVar(&defaults.Host, "host", "0.0.0.0|deprecated", "")
```

- in `config.ensureParsed` check whether the value is equal to the fake one, show warning and restore it back to `"0.0.0.0"`
- in the help show `"default: 0.0.0.0 (deprecated, will be changed to "localhost" )`

Reply:
Sounds good to me. I was waiting for this suggestion.